climpred/bootstrap.py

```diff
@@ -23,26 +23,27 @@
     shift_cftime_singular,
 )
 
+concat_kwargs = {'coords': 'minimal', 'compat': 'override'}
 
-def _resample(hind, resample_dim, to_be_resampled):
+
+def _resample(hind, resample_dim):
     """Resample with replacement in dimension `resample_dim` from values of
-    `to_be_resampled`
+    `to_be_resampled`.
```

Comment:
You can just make this "Resample with replacement in dimension ``resample_dim``." Since you removed that input. FYI, for sphinx/rst you use two `` `` to get the code look on our docs.